skbeam/core/accumulators/binned_statistic.py

```diff
@@ -434,36 +458,26 @@ def __init__(self, rbins, phibins, rowsize, colsize,
                 will be called on the values in each bin.  Empty bins will be
                 represented by function([]), or NaN if this returns an error.
         """
+        if origin is None:
+            origin = shape[0]//2, shape[1]//2
```

Comment:
I mean in general I don't have any issue with it.  I was mostly curious as to how frequently that issue would be hit.  

nit: you don't need the `.` after the `2` since we are importing `division` from `__future__` which makes division with `/` always do float division